packages/hash/api/src/db/postgres/link/getEntityIncomingLinks.ts

```diff
@@ -15,7 +15,7 @@ export const getEntityIncomingLinks = async (
   const rows = await conn.any(sql<DBLinkRow>`
     select ${mapColumnNamesToSQL(linksColumnNames, "links")}
     from links inner join incoming_links on (
-      links.account_id = incoming_links.link_account_id
+      links.source_account_id = incoming_links.source_account_id
```

Comment:
This was using outdated column names of the links table

Reply:
I ran into this with the previous search functionality. Thank you for fixing it :)